src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -153,15 +179,26 @@ private async ValueTask SynchronizePrimaryWorkspaceAsync(CancellationToken cance
         }
     }
 
+    private async ValueTask SynchronizeActiveDocumentAsync(CancellationToken cancellationToken)
+    {
+        var activeDocument = _documentTrackingService.TryGetActiveDocument();
+
+        var client = await RemoteHostClient.TryGetClientAsync(_workspace, cancellationToken).ConfigureAwait(false);
+        if (client == null)
+            return;
+
+        var solution = _workspace.CurrentSolution;
+        await client.TryInvokeAsync<IRemoteAssetSynchronizationService>(
+            (service, cancellationToken) => service.SynchronizeActiveDocumentAsync(activeDocument, cancellationToken),
```

Comment:
note: this call doesn't use CurrentSolution, or pass over any solution data that needs to sync.  It's solely about passing along the current active doc id.